### DISCUSSION TITLE
Wiring for Antlab 3d touch?

### DISCUSSION THREAD

**Discussion by olskar** [Jyers/Marlin#941] [Q&A] [answered]
Hello!

Sorry if this question is stupid but read below in a tutorial and want to be sure I plug it in correctly :)

> Important: Make sure that whatever wiring instructions you follow are meant for your firmware. Between two firmware versions, certain wires may be swapped, so following the incorrect pinout configuration could result in damaging effects

**Comment by mostlyflawles** [comment]
Hard to believe it could differ between firmware, understandably if it would be different board revision.
If I recall correctly my BLtouch was plug and play, couldn't install it any other way.
If your Bltouch came with a 5 pin connector I don't think you could do wrong.

**Comment by GeoStork** [comment]
If, like me, you didn't purchase the Creality BL Touch kit, this adapter helped me.  [https://www.ebay.com/itm/203476194824](url)

**Comment by smooreace** [comment]
LOLOL

**Comment by Jyers** [comment, marked as answer]
Just make sure it's plugged into the BL Touch port and not the z endstop. Careful with which way it's connected as a switched connector is a very easy way to fry your board. There should be plenty of documentation online for how to plug it into the port.
